### PR TITLE
[Peering] create new api version 2020-01-01-preview

### PR DESCRIPTION
**Pull request by rileymckenna**
Added Registered Asn
Added Legacy Peering filter by Asn
Added filter to peering service countries
Added Sku to peering type
Added type to direct peering enum
Changed how contact details are stored
Added Sku to peering service

[Awaiting PR Merge](https://github.com/Azure/azure-rest-api-specs/pull/8140)

### CODE REVIEW

**Comment by rileymckenna** [comment]
SPEC PR merged

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj [hunk 1]
```diff
@@ -7,22 +7,17 @@
 		<PackageId>Microsoft.Azure.Management.Peering</PackageId>
 		<Description>Microsoft Azure Management Peering library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Peering</AssemblyName>
-		<Version>1.0.1-preview</Version>
-		<PackageTags>Azure Peering;</PackageTags>
+		<Version>2.0.0-preview</Version>
+		<PackageTags>Azure Peering;Peering_2020-01-01-preview;</PackageTags>
```
Comment (isra-fel) [inline sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj]: Tags of API version should be in your module's `AzSdk.RP.props` file

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj [hunk 2]
```diff
@@ -7,22 +7,17 @@
...
     <PackageReleaseNotes>
-	<![CDATA[New features:   Adding Peering Service operations. ]]>
+	<![CDATA[New features:   Adding Registered ASN operations. ]]>
```
Comment (isra-fel) [inline sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj]: Please list your change logs in one `<![CDATA[` ... `]]>`  block. You can take Network's csproj as an example

Comment (isra-fel) [inline sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj]: Please list your change logs in one `<![CDATA[` ... `]]>`  block. You can take Network's csproj as an example

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Peering Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Peering management functions for managing the Microsoft Azure Peering service.")]
 
-[assembly: AssemblyVersion("0.12.0.0")]
-[assembly: AssemblyFileVersion("0.12.0.0")]
+[assembly: AssemblyVersion("0.13.0.0")]
```
Comment (isra-fel) [inline sdk/peering/Microsoft.Azure.Management.Peering/src/Properties/AssemblyInfo.cs]: Use the same version as in csproj
```suggestion
[assembly: AssemblyVersion("2.0.0.0")]
```

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -7,8 +7,8 @@
...
+[assembly: AssemblyFileVersion("0.13.0.0")]
```
Comment (isra-fel) [inline sdk/peering/Microsoft.Azure.Management.Peering/src/Properties/AssemblyInfo.cs]:
```suggestion
[assembly: AssemblyFileVersion("2.0.0.0")]
```

File: src/SDKs/_metadata/peering_resource-manager.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+2020-01-23 19:30:24 UTC
```
Comment (isra-fel) [inline src/SDKs/_metadata/peering_resource-manager.txt]: This file should be in `/eng/mgmt/mgmtmetadata/`
Please make sure you generate your code by running `generate.ps1` (not autorest directly). And if the problem still goes on, try [updating build tools](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools)

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @rileymckenna please see my inline comments. Thanks!

**Review by isra-fel — APPROVED** [review]

**Pull request merged**